test/fixtures.py

```diff
@@ -239,20 +269,30 @@ def __init__(self, host, port, broker_id, zk_host, zk_port, zk_chroot,
         self.child = None
         self.running = False
 
+        self._client = None
+
+    def bootstrap_server(self):
+        return '%s:%d' % (self.host, self.port)
+
     def kafka_run_class_env(self):
         env = super(KafkaFixture, self).kafka_run_class_env()
         env['LOG_DIR'] = os.path.join(self.tmp_dir, 'logs')
         return env
 
     def out(self, message):
-        log.info("*** Kafka [%s:%s]: %s", self.host, self.port or '(auto)', message)
+        log.info("*** Kafka [%s:%s]: %s", self.host, self.port or '(auto)', message.decode('utf-8') if isinstance(message, bytes) else message)
```

Comment:
@dpkp do we want to know the actual bytes here for debugging purposes?

Reply:
mmm... I hadn't thought of that. It makes sense to have the raw bytes, rather than a nicely formatted message. I'll revert this